src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -163,60 +163,31 @@ static inline bool is_class_loader(const Symbol* class_name,
 // private: called to verify that k is a static member of this nest.
 // We know that k is an instance class in the same package and hence the
 // same classloader.
-bool InstanceKlass::has_nest_member(InstanceKlass* k, TRAPS) const {
+bool InstanceKlass::has_nest_member(InstanceKlass* k, JavaThread* current) const {
   assert(!is_hidden(), "unexpected hidden class");
   if (_nest_members == NULL || _nest_members == Universe::the_empty_short_array()) {
     if (log_is_enabled(Trace, class, nestmates)) {
-      ResourceMark rm(THREAD);
+      ResourceMark rm(current);
```

Comment:
Since THREAD is only used for ResourceMark and HandleMark which don't have to be a JavaThread, this parameter can just be Thread* current.  And it should be in the first argument so it's not confused with CHECK.

Reply:
I can switch the argument order - that is a good suggestion and avoids the need for other changes. But I will still make it a JavaThread as only a JavaThread executes this code.